components/sst_importer/src/sst_importer.rs

```diff
@@ -145,7 +296,14 @@ impl ImportDir {
         let path = self.join(meta)?;
         let cf = meta.get_cf_name();
         prepare_sst_for_ingestion(&path.save, &path.clone)?;
-        validate_sst_for_ingestion(db, cf, &path.clone, meta.get_length(), meta.get_crc32())?;
+        let length = meta.get_length();
+        let crc32 = meta.get_crc32();
+        if length != 0 || crc32 != 0 {
```

Comment:
We always set the length to 0 in backup & restore then the validation will never be executed, is it expected?

Reply:
Yes it is expected (need to set both length and crc32 to 0), because we decided not to revalidate the SST in BR.

Reply:
Ping @5kbpers ?